.github/workflows/publish-package-docs.yaml

```diff
@@ -0,0 +1,96 @@
+name: Ansible package docs build
+on:
+  workflow_dispatch:
+    inputs:
+      github_fork:
+        description: 'GitHub Fork'
+        required: true
+        default: 'ansible'
+      github_branch:
+        description: 'GitHub Branch'
+        required: true
+        default: 'devel'
+      language:
+        type: choice
+        description: 'Language'
+        required: true
+        default: 'english'
+        options:
+        - 'english'
+        - 'japanese'
+      package_version:
+        type: choice
+        description: 'Ansible Version'
+        required: true
+        default: 'devel'
+        options:
+        - 'devel'
+        - '8'
+        - '7'
+        - '6'
+        - '5'
+        - '4'
+        - '3'
+        - '2.10'
+      latest_symlink:
+        type: boolean
+        description: 'Add latest symlink'
+        required: true
+
+jobs:
+  build-package-docs:
+    runs-on: self-hosted
+    env:
+      PACKAGE_VERSION: "${{ github.event.inputs.package_version }}"
+      LANGUAGE: ${{ github.event.inputs.language}}
+    steps:
+      - name: Checkout Ansible
+        uses: actions/checkout@v3
+        with:
+          repository: ${{ github.event.inputs.github_fork }}/ansible
+          ref: ${{ github.event.inputs.github_branch }}
+          path: build-directory
+
+      - name: Output Python info
+        run: python --version --version && which python
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install setuptools six wheel
+
+      - name: Install project requirements
+        run: |
+          pushd build-directory
+          python -m pip install -r requirements.txt
+          python -m pip install -r docs/docsite/requirements.txt
```

Comment:
fwiw - we shouldn't long-term use docs/docsite/requirements.txt for anything but the devel docs build.

Reply:
interesting. this is in the current jenkins job. why shouldn't we use it?